cmd/statusd/main.go

```diff
@@ -54,6 +55,7 @@ var (
 	pprofPort         = flag.Int("pprofport", 52525, "Port for runtime profiling via pprof")
 	logLevel          = flag.String("log", "INFO", `Log level, one of: "ERROR", "WARN", "INFO", "DEBUG", and "TRACE"`)
 	logFile           = flag.String("logfile", "", "Path to the log file")
+	logWithoutColors  = flag.Bool("log-without-colors", false, "Disables log colors")
```

Comment:
I would put singular instead `Disable log colors` to keep consistent with other options